CMakeLists.txt

```diff
@@ -240,42 +240,24 @@ endif()
 
 export_components("${CMAKE_BINARY_DIR}/components.ini")
 
+# it seems like cmake cannot uncompress *.gz files... only ZIP/tar.gz.
+# Lets download the uncompressed versions. 
+# https://gitlab.kitware.com/cmake/cmake/-/issues/23054
+
 set(PCI_IDS_FILE pci.ids)
-set(PCI_IDS_GZ_URL https://pci-ids.ucw.cz/v2.2/${PCI_IDS_FILE}.gz)
-set(PCI_IDS_GZ_PATH ${CMAKE_BINARY_DIR}/${PCI_IDS_FILE}.gz)
+set(PCI_IDS_URL https://pci-ids.ucw.cz/v2.2/${PCI_IDS_FILE})
 set(PCI_IDS_PATH ${CMAKE_BINARY_DIR}/${PCI_IDS_FILE})
 set(PCI_IDS_INSTALL_PATH ${CMAKE_INSTALL_DATAROOTDIR}/${PCI_IDS_FILE})
 
-if(ENABLE_PCI_IDS_DOWNLOAD AND NOT EXISTS ${PCI_IDS_GZ_PATH})
-    download_file("${PCI_IDS_GZ_URL}" "${PCI_IDS_GZ_PATH}")
-endif()
-
-if(EXISTS ${PCI_IDS_GZ_PATH} AND NOT EXISTS ${CMAKE_STAGING_PREFIX}/${PCI_IDS_INSTALL_PATH})
-    message(STATUS "Extracting PCI ID database from ${PCI_IDS_GZ_PATH}...")
-    file(MAKE_DIRECTORY "${CMAKE_STAGING_PREFIX}/${CMAKE_INSTALL_DATAROOTDIR}")
-    if (CMAKE_VERSION VERSION_LESS 3.18.0)
-        execute_process(COMMAND "${GZIP_TOOL}" -d -c "${PCI_IDS_GZ_PATH}" OUTPUT_FILE "${CMAKE_STAGING_PREFIX}/${PCI_IDS_INSTALL_PATH}")
-    else()
-        file(ARCHIVE_EXTRACT INPUT "${PCI_IDS_GZ_PATH}" DESTINATION "${PCI_IDS_GZ_PATH}" PATTERNS "${CMAKE_STAGING_PREFIX}/${PCI_IDS_INSTALL_PATH}")
-    endif()
+if(ENABLE_PCI_IDS_DOWNLOAD AND NOT EXISTS ${CMAKE_STAGING_PREFIX}/${PCI_IDS_INSTALL_PATH} )
+    download_file("${PCI_IDS_URL}" "${CMAKE_STAGING_PREFIX}/${PCI_IDS_INSTALL_PATH}")
```

Comment:
I'm not sure about the assertion in the commit message that downloading 2 MiB of data vs 500 KiB is "meaningless". On my internet connection, it doesn't matter, but there's contributors with much less powerful internet connections.

Reply:
The last update to serenityOS's git (`git pull`) gave me these numbers:
```
remote: Total 3080 (delta 2417), reused 2379 (delta 2350), pack-reused 529
Receiving objects: 100% (3080/3080), 856.94 KiB | 369.00 KiB/s, done.
```
Which means, that on "every week" we add about 500kb of git metadata. 

Then on build we download 44M from CLDR.